### PR TITLE
Improving test coverage

### PR DESCRIPTION
**Pull request by v-kerimov**
82% coverage, according to my IDE's tool.

### CODE REVIEW

**Comment by o-kopysov** [comment]
@v-kerimov 
Please fix all build warnings.

File: pom.xml [hunk 1]
```diff
@@ -115,15 +120,14 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.10.1</version>
                 <configuration>
-                    <release>11</release>
```
Comment (o-kopysov) [inline pom.xml:118]: Please return back this line

File: pom.xml [hunk 2]
```diff
@@ -115,15 +120,14 @@
...
                     <showWarnings>true</showWarnings>
                     <showDeprecation>true</showDeprecation>
                     <compilerArgs>
-                        <arg>-Xlint:all,-serial,-processing</arg>
```
Comment (o-kopysov) [inline pom.xml:122]: Please return back this line

File: src/main/java/com/lpvs/entity/ResponseWrapper.java [hunk 3]
```diff
@@ -8,10 +8,17 @@
...
         this.message = message;
     }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
```
Comment (o-kopysov) [inline src/main/java/com/lpvs/entity/ResponseWrapper.java:23]: Do we need these methids?

**Review by o-kopysov — APPROVED** [review]
LGTM

**Comment by o-kopysov** [comment]
@MoonkiHong  @t25kim 
Please take a look at this PR.

**Review by MoonkiHong — APPROVED** [review]
+1

**Pull request merged**